tests/mysql/mysql_integration_test.go

```diff
@@ -491,3 +498,130 @@ func runMySQLListActiveQueriesTest(t *testing.T, ctx context.Context, pool *sql.
 	}
 	wg.Wait()
 }
+
+func runMySQLListTableFragmentationTest(t *testing.T, tableNameParam, tableNameAuth string) {
```

Comment:
```suggestion
func RunMySQLListTableFragmentationTest(t *testing.T, tableNameParam, tableNameAuth string) {
```

Can you please add this to `tool.go` right under `RunMySQLListActiveQueriesTest()`, and add this test to the `cloud_sql_mysql_integration_test.go` too?